tests/data/meta.json

```diff
@@ -1 +1 @@
-{"latitude": 39.7400016784668, "longitude": -105.16999816894531, "altitude": 1782, "timezone": -7.0, "country": "United States", "state": "Colorado", "county": "Jefferson"}
+{"latitude": 39.7400016784668, "longitude": -105.16999816894531, "altitude": 1782, "timezone": -7.0, "country": "United States", "state": "Colorado", "county": "Jefferson", "Wind_Height_m": 2}
```

Comment:
I think for the tests it would be good practice to keep the behavior of the input files as is compared to current versions of NSRDB. This will help people debug their code.

Reply:
I changed the get_weather function to look at where the data came from and insert meta data for the wind speed measurement height "Wind_Height_m". Then when you go to calculate the module temperature you use what ever wind speed power factor relating wind speed and height that is appropriate for that temperature model. If a data set doesn't have this factor, the code can manage it through checks.

I'd suggest to leave it as I did it.

Reply:
if the checks are in place, shouldn't the get_weather function be able to load the original dataset (without the wind height) and include the wind heigth during loading/preparation of the data?

Reply:
also here you can see the lowercase naming convention from NSRDB/pvlib

Reply:
the wind_height is not included in the satellite data and is just added based on the knowledge that the NSRDB uses 2 m and PVGIS uses 10m. So I guess your question is, should this correction be done during the temperature calculation as opposed to when the data is obtained from the satellite database. I prefer to have it earlier because it is a fundamental aspect of the dataset that was neglected by the developers. Also, it is possible that for some other purpose besides module temperature estimates that someone would want this height so having it put in upfront reduces the amount of overall coding that is necessary and should eliminate errors.

Yes, we need to go over the whole thing and fix the capitals in the variables.

Reply:
I adapted the naming convention. We can keep it in the meta.json for the tests.